README.md

```diff
@@ -33,6 +33,10 @@ The container expects the following environment variables:
 
 `PRIVATE_KEY` - Private key content
 
+`PASSWORD`  - Password in case you do not use a private key (not recommended)
+
+`PASSPHRASE` - Passphrase to your private key
```

Comment:
Should be not mandatory.

Reply:
empty should work - I however did not explicitly test it without